src/test/java/com/google/sps/integration/ElectionInfoIT.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.integration;
+
+import org.junit.*;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class ElectionInfoIT {
```

Comment:
Can we leave out this file for now since there are no tests inside of it?

Reply:
yup -- I'll go ahead and remove it now.